### ISSUE TITLE
Custom encrypted suffix

### ISSUE THREAD

**Issue by youtehub** [rclone/rclone#6932]
#### What is your current rclone version (output from `rclone version`)?
[rclone v1.62.2](https://github.com/rclone/rclone/releases/tag/v1.62.2)

#### What problem are you are trying to solve?

The current encryption suffix is *.bin, I hope there is an option to customize the encrypted file suffix, such as changing to *.jpg, *.mp4, which is to customize the suffix to disguise the encrypted file.
In this way, rclone encrypted files can be placed in different places according to user needs

#### How do you think rclone should be changed to solve that?

In the encrypted crypt configuration file of rclone, an optional configuration item **custom_suffix** is provided to allow users to customize, and the default is still in the form of **bin**

Custom encrypted suffixes are not yet supported, the following example is a sample configuration file template given by myself
```
[folderCrypt]
type = crypt
remote = D:\folders
directory_name_encryption = false
password = password1
password2 = password2 
filename_encoding = base32768
filename_encryption = standard
custom_suffix = mp4
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by albertony** [comment]
In my mind that makes sense, and don't think it would be that much work to make the following into an option:

https://github.com/rclone/rclone/blob/c502e00c87854781a939b385dda0e9d38ecbd81f/backend/crypt/cipher.go#L41

**Comment by ncw** [comment]
Yes it could be an option if someone wants to have a go at it.

**Comment by albertony** [comment]
Oh, and then allowing none / empty string should probably be allowed, in cases where path length is critical..

**Comment by ncw** [comment]
I think this is all done now in 281a007b1a7867db5904ee09131e5809c0606f3d ghank you @jladbrook 

Currently in [the latest beta](https://beta.rclone.org/) and to be released in v1.63

**Comment by 88elie88** [comment]
Feature request: preserve suffix on encrypted names (filename_encryption = standard)
What problem are you trying to solve?
I use multiple crypt remotes on the same underlying storage, all sharing the
same password, to fit two different workflows:
Local device: filename encryption off (I need to recognize files by
name day-to-day), but file content encrypted.
Cloud backup: filename encryption standard (full anonymity, since
the cloud is less trusted), and content also encrypted.
Because both remotes share the same password, I set up a third "combined"
crypt remote (both name+content encryption) so I can go directly from the
cloud-style encrypted file back to the local-style file in one pass,
instead of decrypting through each layer separately.
The problem: my local remote (filename_encryption=off) adds a suffix (e.g.
.rc) via --crypt-suffix, so a local file becomes photo.jpg.rc. When that
file passes through the remote with filename_encryption=standard, the suffix
is NOT treated as separate — the entire string photo.jpg.rc gets encrypted
as one block. --crypt-suffix on that remote has no effect at all, because
(per source) it's only applied in the NameEncryptionOff branch.
So when I decrypt back through the "combined" remote, I get back
photo.jpg.rc — correct, but only because the suffix survived by accident
inside the encrypted blob, not because rclone preserved it as a suffix. This
makes it impossible to strip/manage the suffix in a predictable, automatic
way when chaining crypt layers.
How do you think rclone should be changed to solve that?
Allow --crypt-suffix to apply even when filename_encryption = standard,
appended after the encrypted name (e.g. a8k2j...xyz.rc — a fixed, generic
suffix, not derived from the real file extension).
To avoid any backward-compatibility risk, the default suffix in this mode
could simply be none (i.e. current behaviour, no change) unless the user
explicitly sets a suffix. That way existing setups and already-encrypted
files are completely unaffected, and only users who opt in by setting
--crypt-suffix get the new behaviour.
This would let users:
Tag encrypted files by "which crypt layer/origin produced them" without
leaking any real file-type information (the suffix is generic/fixed).
Fix compatibility issues with tools like tar/zip that sometimes fail on
extensionless encrypted names (see #6898).
Enable reliable suffix stripping when chaining crypt remotes with different
filename_encryption settings but the same password (see #6932 for the
related "custom suffix" request, already labeled enhancement/help wanted).
This is related to #6932 and #6898, but specifically scoped to the
"standard" filename encryption mode rather than "off".
Also related in spirit: #5740 requests the opposite asymmetry (directory
names encrypted while file names stay in clear), which is currently blocked
because directory_name_encryption is documented to do nothing when
filename_encryption = off. Both requests point to the same underlying gap:
crypt's per-component settings (filename / dirname / suffix) are more
tightly coupled to filename_encryption mode than they need to be, which
limits mixing encrypted and cleartext components for setups like mine that
rely on multiple crypt layers with a shared password.
Would be happy to see this reopened given the (simple, backward-compatible)
suggested fix above.
